### DISCUSSION TITLE
macos high sierra 10.13.6,

### DISCUSSION THREAD

**Discussion by iamjfk11** [transmission/transmission#5427] [General]
### What is the issue?

after the last 2 or 3 updates, the client seems to ignore the setting, bind-to-address ipv4, orm.m0k.transmission, seems correct, needless to say that the documentation about it is sparse, of course thank you and congratulations because client, it's simple, but simply it's also great
 i use this command "defaults write org.m0k.transmission bind-address-ipv4 192.168.2.211"
this "help" = https://trac.transmissionbt.com/wiki/EditingConfigurationParameters
it is not clear, "numeric" "string" or "boolean", yes but the ":" the "quotes", i don't understand anything, or rather i understand but not clearly without doubt
(my net is only ipv4)

thx   bye

Filippo

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.3

**Comment by dantearmok** [comment]
defaults write org.m0k.transmission bind-address-ipv4 "192.168.2.211"

Why do you need to do this?
